src/argumentDefinitions.ts

```diff
@@ -0,0 +1,15 @@
+import { FragmentDefinitionNode, visit, ArgumentNode } from "graphql"
```

Comment:
Import `visit` from `./dependencies` to ensure we never mix types from vscode-apollo’s and the user’s project.